operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -66,16 +70,17 @@ public static void after() {
     public void testSuccessfulReconcile(VertxTestContext context)  {
         MetricsProvider metrics = createCleanMetricsProvider();
 
-        AbstractWatchableStatusedResourceOperator resourceOperator = resourceOperatorWithExistingResource();
+        FooWatchableStatusedResourceOperator resourceOperator = resourceOperatorWithExistingResource();
+
+        FooOperator operator = new FooOperator(vertx, "TestResource", resourceOperator, metrics, null) {
```

Comment:
Why do we need to rename these?

Reply:
I created a specific type or 2 to handle all the raw type warnings (and when I added the actual type parameters, the code became unreadable). But on my second pass I'll probably suppress raw types for the class and leave it at that.